pkg/daemons/agent/agent.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/rancher/k3s/pkg/daemons/executor"
 	"github.com/rancher/k3s/pkg/util"
 	"github.com/rancher/k3s/pkg/version"
+	"github.com/rootless-containers/rootlesskit/pkg/parent/cgrouputil"
```

Comment:
Maybe, we should have a comment line to explain that this logic is not specific to rootless

Reply:
:heavy_check_mark: